data/tribes/initialization/empire/units.lua

```diff
@@ -1129,6 +1129,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_headquarters",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Headquarters
+            lore = pgettext("empire_building", "‘Omnes viæ ducunt Fremilem.’ (All roads lead to Fremil)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Headquarters
+            lore_author = pgettext("proverb signifying the glory of the Empire and its capital"),
```

Comment:
Use sentence case for the lore authors:
```suggestion
            lore_author = pgettext("Proverb signifying the glory of the Empire and its capital"),
```
more below